roles/pulp_devel/tasks/config_files.yml

```diff
@@ -32,19 +32,30 @@
     dest: "{{ developer_user_home }}/.config/pulp_smash/settings.json"
     force: yes
     mode: 0600
+- name: Create Trust Pulp CA temporary script
```

Comment:
You need `changed_when: false` on the create and the remove tasks.

Or better yet, just put this on all 3 tasks:
```
  tags:
    - molecule-idempotence-notest
```